fendermint/vm/actor_interface/src/ipc.rs

```diff
@@ -64,6 +64,18 @@ lazy_static! {
                             name: "XnetMessagingFacet",
                             abi: ia::xnet_messaging_facet::XNETMESSAGINGFACET_ABI.to_owned(),
                         },
+                        EthFacet {
+                            name: "DiamondLoupeFacet",
+                            abi: ia::diamond_loupe_facet::DIAMONDLOUPEFACET_ABI.to_owned(),
+                        },
+                        EthFacet {
+                            name: "DiamondCutFacet",
+                            abi: ia::diamond_cut_facet::DIAMONDCUTFACET_ABI.to_owned(),
+                        },
+                        EthFacet {
+                            name: "OwnershipFacet",
+                            abi: ia::ownership_facet::OWNERSHIPFACET_ABI.to_owned(),
+                        },
```

Comment:
We were missing these facets from the gateway. Although the owner address will be f01 (Init actor), as that's the actor that creates the gateway and the registry during genesis:

https://github.com/consensus-shipyard/ipc/blob/f9bda9e83dcd6c7175a25df604f16aec40ffa973/fendermint/vm/interpreter/src/fvm/state/genesis.rs#L434

Which means that we anyway can't upgrade the gateway and registry actors inside a subnet without a hard fork.

cc @fridrik01 @cryptoAtwill